app/index.html

```diff
@@ -27,27 +27,23 @@
     <!-- endbuild -->
 
     <!-- build:js static/scripts/angular.js -->
-    <script src="bower_components/angular/angular.js"></script>
-    <script src="bower_components/angular-route/angular-route.js"></script>
-    <script src="bower_components/angular-cookies/angular-cookies.js"></script>
-    <script src="bower_components/angular-truncate-2/src/angular-truncate-2.js"></script>
-    <script src="bower_components/angular-sanitize/angular-sanitize.js"></script>
+    <script src="bower_components/angular/angular.min.js"></script>
```

Comment:
I assumed the minified files would be concatenated, because it worked properly when I tested  after building the app,  but I'm not sure. 

I will remove the min dependencies. But I think it's best to use the official minified versions, i will work on a solution in the free sprint.